examples/random.roc

```diff
@@ -53,18 +53,27 @@ render = \model, { keys, timestampMillis } ->
 
     { seed, lines } = randomList model.seed generator model.number
 
-    Task.forEach! lines RocRay.drawRectangle
+    (forEach! lines RocRay.drawRectangle!) {}
 
     RocRay.drawText! { pos: { x: 10, y: model.height - 25 }, text: "Up-Down to change number of random dots, current value is $(Num.toStr model.number)", size: 20, color: White }
 
-    RocRay.endDrawing!
+    RocRay.endDrawing! {}
 
     if Keys.down keys KeyUp then
-        Task.ok { model & seed, number: Num.addSaturated model.number 10 }
+        Ok { model & seed, number: Num.addSaturated model.number 10 }
     else if Keys.down keys KeyDown then
-        Task.ok { model & seed, number: Num.subSaturated model.number 10 }
+        Ok { model & seed, number: Num.subSaturated model.number 10 }
     else
-        Task.ok { model & seed }
+        Ok { model & seed }
+
+# not sure this is ok, but just trying to replace Task.forEach
+forEach! : List a, (a => {}) => ({} => {})
+forEach! = \things, do -> \{} ->
```

Comment:
This doesn't need to return a thunk, it can just be:

```roc
forEach! : List a, (a => {}) => {}
forEach! = \list, do! ->
    when list is
        [] -> {}
        [x, .. as xs] ->
            do! x
            forEach! xs do!
```